shell/platform/android/io/flutter/view/FlutterMain.java

```diff
@@ -225,6 +227,7 @@ private static void initResources(Context applicationContext) {
             .addResource(sAotIsolateSnapshotData)
             .addResource(sAotIsolateSnapshotInstr)
             .addResource(sFlx)
+            .addResource(sSnapshotBlob)
```

Comment:
Should probably add `sSnapshotBlob` to `initConfig()` above, so we can override the name in the metadata, if we want.

Reply:
Yes and then add it switches.h?  On Android we have the option to pass in the snapshot through an intent. Should we still keep that then? I'll wait with this a bit to see if there are more comments.

Reply:
Yep, we'll definitely need support for it in `switches.h` (and `initConfig()`).  the reason is that inside Google's Bazel-based build system, each build artifact needs to be traceable back to a unique build rule, which in turn means that two build rules in the same package may not produce build artifacts with the same name.  The way we accomplish this is we derive the artifact names from the build rule names.

Currently, always using `snapshot_blob.bin` is ok because it's not a direct build artifact -- it's zipped inside an FLX artifact whose name is unique.  But once we remove FLX, we'll need to control the name of the snapshot in the build system and them pass that through to the engine the way we do with `isolate_snapshot_data` and `isolate_snapshot_instr`.